internal/installcfg/installcfg.go

```diff
@@ -292,8 +277,9 @@ func (i *installConfigBuilder) setBMPlatformInstallconfig(cluster *common.Cluste
 		if swag.StringValue(host.Status) == models.HostStatusDisabled {
 			continue
 		}
-		i.log.Infof("host name is %s", hostutil.GetHostnameForMsg(host))
-		hosts[yamlHostIdx].Name = i.getBMHName(host, &masterIdx, &workerIdx)
+		hostName := hostutil.GetHostnameForMsg(host)
```

Comment:
Why are we using GetHostnameForMsg and not GetCurrentHostName ?

Reply:
`GetHostnameForMsg` is calling `GetCurrentHostName` and adds additional handling in case the latter did not return a valid name. So I picked `GetHostnameForMsg` because it is a safer version of `GetCurrentHostName`

https://github.com/openshift/assisted-service/blob/master/internal/host/hostutil/host_utils.go#L32
https://github.com/openshift/assisted-service/blob/master/internal/host/hostutil/host_utils.go#L20